src/pages/experiments/[experimentId]/plots-and-tables/marker-heatmap/index.jsx

```diff
@@ -99,11 +99,13 @@ const MarkerHeatmap = ({ experimentId }) => {
       // grouping and metadata tracks should change when selectedCellSet is changed
       updatesToDispatch = {
         ...updatesToDispatch,
-        selectedTracks: [config.selectedCellSet],
-        groupedTracks: [config.selectedCellSet],
+        selectedTracks: [updatesToDispatch.selectedCellSet],
+        groupedTracks: [updatesToDispatch.selectedCellSet],
       };
     }
 
+    dispatch(updatePlotConfig(plotUuid, updatesToDispatch));
```

Comment:
Moved this so that `loadDownsampledGeneExpression`, which uses the config from redux has the updated redux config.

This didn't seem to be a problem for some reason when I tested, but just in case there's the danger of a race condition, doing it like this guarantees that redux has the updated form when we send the subsequent requests